cmd/influxd/main.go

```diff
@@ -45,6 +35,10 @@ func main() {
 		commit = "unknown"
 	}
 
+	// Set parallelism.
+	runtime.GOMAXPROCS(runtime.NumCPU())
+	log.Printf("GOMAXPROCS set to %d", runtime.NumCPU())
```

Comment:
So I actually think it should be `log.Printf("GOMAXPROCS set to %d", runtime.GOMAXPROCS(0))`. When the value is set to anything < 1, nothing is changed and you get what `GOMAXPROCS` actually is. Just so the log message is absolutely correct.